### PR TITLE
PS-5174 Add missing test cases for compressed columns  with partition

### PR DESCRIPTION
**Pull request by altmannmarcelo**
Created a common test case to validate compressed columns with
partitioned tables for tokudb and myrocks.

### CODE REVIEW

**Comment by altmannmarcelo** [comment]
https://ps.cd.percona.com/view/5.7/job/percona-server-5.7-param/133/

**Comment by altmannmarcelo** [comment]
Related test cases:

```
rocksdb.partition 'write_committed'      w2 [ pass ]   1171
rocksdb.partition 'write_committed-native_partitioning' w4 [ pass ]   1152
rocksdb.partition 'write_prepared'       w1 [ pass ]   1257
tokudb.parts.partition_special_tokudb 'non-native-partitioning' w3 [ pass ]  27353
tokudb.parts.partition_special_tokudb 'native-partitioning' w2 [ pass ]  27262
```

File: mysql-test/include/partition_with_compressed_columns.inc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# ==== Purpose ====
```
Comment (laurynas-biveinis) [inline mysql-test/include/partition_with_compressed_columns.inc]: Add percona_ prefix to the test file

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: mysql-test/include/partition_with_compressed_columns.inc (hunk 1 above)
Comment (altmannmarcelo) [reply mysql-test/include/partition_with_compressed_columns.inc]: @laurynas-biveinis ready for next review

File: mysql-test/include/percona_partition_with_compressed_columns.inc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# ==== Purpose ====
+#
+# Common test to check support for compressed columns on partitioned tables
+# On multiple Storage Engines.
+#
+# ==== Usage ====
+#
+# --let $engine= ENGINE
+# --source include/partition_with_compressed_columns.inc
```
Comment (laurynas-biveinis) [inline mysql-test/include/percona_partition_with_compressed_columns.inc]:
```suggestion
# --source include/percona_partition_with_compressed_columns.inc
```

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: mysql-test/include/percona_partition_with_compressed_columns.inc (hunk 2 above)
Comment (altmannmarcelo) [reply mysql-test/include/percona_partition_with_compressed_columns.inc]: @laurynas-biveinis good catch. Ready for next review

**Review by laurynas-biveinis — APPROVED** [review]

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by laurynas-biveinis** [comment]
@altmannmarcelo, please always merge approved PRs in full sets (i.e. when 8.0 version is approved too) unless agreed otherwise

**Pull request merged**